### DISCUSSION TITLE
Web Components e Testes Unitários

### DISCUSSION THREAD

**Discussion by isabelandss** [frontendbr/forum#1662] [Discussão]
Olá, galera!

Eu estou estudando web components e estou tendo dificuldades em encontrar material de testes unitários para web components. Alguém sabe onde posso encontrar? Se quiseram indicar materiais bons para web components fico muito grata!

Obrigada!

**Comment by portothree** [comment]
Já deu uma olhada no [web-component-tester](https://github.com/Polymer/tools/tree/master/packages/web-component-tester)? O slack da Polymer também tem algumas questões legais sobre testes unitários.

**Comment by developerfred** [comment]
Você pode usar as libs existentes para testar os componentes aqui uma pequena lista de algumas bibliotecas de test para frontend:

## Unit Test
- [Jasmine](https://jasmine.github.io)
- [Jest](https://jestjs.io)
- [Ava](https://github.com/avajs/ava)

# e2e - End To End
- [Karma](https://karma-runner.github.io/latest/index.html)
- [Cypress](https://www.cypress.io)
